types/V1.mli

```diff
@@ -102,6 +102,46 @@ module type CLOCK = sig
   (** Convert a time in seconds, as returned by {!time}, into a date
       and a time. Assumes UTC (Coordinated Universal Time), also known
       as GMT. *)
+
+end
+
+(** {1 Posix clock}
+
+    Clock returning time since the unix epoch. Subject to adjustment by eg. NTP.
+    Compatible with the Ptime library. *)
+module type PCLOCK = sig
+
+  val now_d_ps : unit -> int * int64
+  (** [now_d_ps ()] is [(d, ps)] representing the POSIX time occuring
+      at [d] * 86'400e12 + [ps] POSIX picoseconds from the epoch
+      1970-01-01 00:00:00 UTC. [ps] is in the range
+      \[[0];[86_399_999_999_999_999L]\]. *)
+
+  val current_tz_offset_s : unit -> int option
+  (** [current_tz_offset_s ()] is if available the clock's current local time zone
+      offset to UTC in seconds. *)
```

Comment:
I think this should be placed by what I mentioned in [this comment](https://github.com/mirage/mirage/issues/442#issuecomment-166742871). That is:
```
(** [current_tz_offset_s ()] is the clock's current local time zone
    offset to UTC in seconds, if known. This is the duration local time -
    UTC time in seconds. *)
```